bin/load-kyuubi-env.sh

```diff
@@ -18,7 +18,7 @@
 
 
 if [ -z "${KYUUBI_HOME}" ]; then
-  export ${KYUUBI_HOME:-"$(cd "$(dirname "$0")"/.. || exit; pwd)"}
+  export KYUUBI_HOME="${KYUUBI_HOME:-"$(cd "$(dirname "$0")"/.. || exit; pwd)"}"
```

Comment:
KYUUBI_HOME is always empty
```suggestion
  export KYUUBI_HOME="$(cd "$(dirname "$0")"/.. || exit; pwd)"
```

Reply:
get it, thanks